packages/story-editor/src/masks/display.js

```diff
@@ -85,6 +114,68 @@ export default function WithMask({
     previewMode ? '-preview' : ''
   }-${randomId}`;
 
+  if (showSingleBorder) {
```

Comment:
If this is exactly the same as in `output.js`,  could be move it to a helper (DRY)?

Reply:
This was a tough refactor (so many things were similar but not exactly the same), but managed it in 1c2a42a8c.